models/base.go

```diff
@@ -276,7 +280,7 @@ func (b *Base) request(ctx context.Context, method string, url string, body []by
 			responseBody = nil
 		}
 	default:
-		success = resp.StatusCode == 200
```

Comment:
I was done this way in collection implementation, decided this is a good change to move in base